src/core/jit/exec.rs

```diff
@@ -70,6 +71,28 @@ where
     async fn init(&mut self) {
         join_all(self.plan.as_children().iter().map(|field| async {
             let ctx = Context::new(&self.request);
+            let mut arg_map = indexmap::IndexMap::new();
+            for arg in field.args.iter() {
+                let name = async_graphql::Name::new(&arg.name);
+                let value: Option<anyhow::Result<ConstValue>> = arg
```

Comment:
`exec` can't be bound to `ConstValue` or any specific value. Use the type param `Value` here.

Reply:
since in `Arg` type field `value` is `async_graphql_value::Value` and `default_value` is `async_graphql_value::ConstValue` so the value of any argument will also have to be `ConstValue`. If we want this to be generic I think we also will have to update `Arg` type to use generic types. Should I update that as well?